aws/credentials/process_provider_test.go

```diff
@@ -0,0 +1,143 @@
+package credentials
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
```

Comment:
The SDK is looking to move away from `testify` and use pure go testing. It would be great to the pure go `testing` package for these tests.